lib/active_model/serializer.rb

```diff
@@ -6,6 +6,8 @@
 require 'active_model/serializer/fieldset'
 require 'active_model/serializer/lint'
 
+# ActiveModel::Serializer is an abstract class that is
+# reified when subclassed to decorate a resource.
```

Comment:
@BenMorganIO No no, "to reify" is a verb that means "to make something abstract concrete" (well, technically, "to *look at* something abstract as a concrete thing"). Here he just means that this is an abstract class, the use of which will be to serve as a base for actual serializers.